### PR TITLE
chore: add optimize translation diff

### PR DESCRIPTION
**Pull request by misiekhardcore**
## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Review by grlimacan — APPROVED** [review]
LGTM!

File: optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md [hunk 1]
```diff
@@ -52,3 +52,13 @@ Optimize now supports Java 21+.
...
+
+## Changes in translation files
+
+### Localization file
+
+The following terms have been added to or removed from the localization file `en.json` since the last release:
+
+[en.json.diff](../translation-diffs/differences_localization_313_314.diff)
+
+- Lines with a `+` in the beginning mark the addition/update of a term; lines with a `-` mark the removal of a term.
```
Comment (mesellings) [inline optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md:64]:
```suggestion
- Lines with a `+` in the beginning mark a term addition/update. Lines with a `-` mark a term removal.
```
Just a minor rewording suggestion, non-blocking 👍

**Review by mesellings — APPROVED** [review]
Lgtm 👍 (just one minor rewording suggestion for conciseness, but you do not need to incorporate)

**Review by mesellings — APPROVED** [review]
Lgtm 👍 🚀

**Pull request merged**